_posts/2024-07-04-cleanup-rawhide-builds.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Rawhide (aka "rolling") chroot EOL policy in Copr
+author: Pavel Raiskup
+layout: post
+---
+
+In Fedora Copr, we strive to avoid wasting disk storage unnecessarily.  Disk
+storage is the commodity that incurs the highest cloud costs.  For example, we
+clean the build results for EOL chroots, we clean source builds (build results
+that are imported into DistGit), we clean the DistGit lookaside cache, build
+logs, uploaded files, failed build logs, and more.  Yet, we maintain >= 40TB of
+data with RAID redundancy, volume snapshots, ...
+
+[Most of the Fedora Copr backend
+storage](https://copr-be.cloud.fedoraproject.org/stats/distro.html) is used for
+holding the build results related to Fedora, with the largest consumer being the
+Fedora Rawhide chroot.
+
+The "problem" with Rawhide chroots is that these are "rolling" ones, which never
+reach EOL (unlike the branched Fedoras; see, for example, the data consumption
+for the old Fedora 35 in the previous link if you want to see the difference).
```

Comment:
I would go with something like

The "problem" with Fedora Rawhide as a "rolling" distribution is that it never reaches EOL, unlike branched Fedora releases. For example, take a look at the data consumption for the old Fedora 35 in the previous link if you want to see the difference.